visualization/app/codeCharta/ui/ribbonBar/ribbonBar.po.ts

```diff
@@ -1,10 +1,11 @@
 export class RibbonBarPageObject {
 	private EXPANDED = "expanded"
+	private EXPANDED_REMOVE = "expanded-remove"
 
 	async isPanelOpen(selector: string) {
 		await page.waitForSelector(`#${selector}-card`)
 		const classNames = await page.$eval(`#${selector}-card`, element => element["className"])
-		return classNames.includes(this.EXPANDED)
+		return classNames.includes(this.EXPANDED) && !classNames.includes(this.EXPANDED_REMOVE)
```

Comment:
Where is the remove class ever set? I can't see where this is added :)

The same applies to the next file.

Reply:
this is generated internally by angular, can be seen in slow mo mode. It's very quick though.